Makefile

```diff
@@ -42,6 +42,14 @@ run-flask: ## Run flask
 run-flask-with-docker: ## Run flask
 	./scripts/run_locally_with_docker.sh api-local
 
+.PHONY: run-local-tests-with-docker
+run-local-tests-with-docker:
+	docker build -f docker/Dockerfile --target concourse_tests -t notifications-api .
+	./scripts/run_locally_with_docker.sh api-local
+	docker exec -it notifications_api /bin/bash -c "service postgresql start"
+	docker exec -it notifications_api make test
```

Comment:
I think there's theoretically a race condition here between the DB coming online and the tests running. I don't hit it locally and I think the risk of hitting it is fairly small given our slow-ish unit test startup times.

Maybe just one to be aware of and address if we start seeing issues.

Reply:
maybe, but i tried a couple (of a hundred) times....LOL no race condition has happened. we could add a sleep if desired.